Gemfile.lock

```diff
@@ -68,8 +68,8 @@ GEM
       popper_js (>= 1.14.3, < 2)
       sassc-rails (>= 2.0.0)
     builder (3.2.4)
-    byebug (11.0.1)
-    capybara (3.30.0)
+    byebug (11.1.1)
+    capybara (3.31.0)
```

Comment:
### [capybara](https://github.com/teamcapybara/capybara)

**Minor** version upgrade :chart_with_upwards_trend::large_orange_diamond: 3.30.0 → 3.31.0

[_[change-log](https://github.com/teamcapybara/capybara/blob/master/History.md), [source-code](https://github.com/teamcapybara/capybara)_]





<details>
<summary>Commits</summary>

A change of **26** commits. See the full changes on [the compare page](https://github.com/teamcapybara/capybara/compare/3.30.0...3.31.0).

These are the first 10 commits:
- (118b3e4) [Fix a Ruby 2.7 deprecation warning.](https://github.com/teamcapybara/capybara/commit/118b3e44142f9ffe53b1727ac92632a30efc8bf6)
- (03e065e) [Merge pull request #2291 from connorshea/fix-27-deprecation](https://github.com/teamcapybara/capybara/commit/03e065e976b9414bae74b1360bc57c53f7bb0b22)
- (a76a35f) [Fix Ruby 2.7 deprecation warnings](https://github.com/teamcapybara/capybara/commit/a76a35fbc8bb5baceb13c50ede1b9ee551f674d7)
- (5d24424) [Fix Chrome download settings](https://github.com/teamcapybara/capybara/commit/5d24424b135be8539810f4e65b7bc920b40ede8f)
- (c5cf02b) [Update History.md for 3.30.1 [ci skip]](https://github.com/teamcapybara/capybara/commit/c5cf02b1a62c1a5b1af0f30b91abc804df7b467f)
- (7eb7e1d) [Implement enabled_options and disabled_options filters for s…](https://github.com/teamcapybara/capybara/commit/7eb7e1deecddc3e735bf9399c027b9639ee0fc7b)
- (c8bceae) [Merge pull request #2292 from teamcapybara/feature_2289](https://github.com/teamcapybara/capybara/commit/c8bceae5ac95b607b58acd723d647deb98e49cc3)
- (4786328) [Fix test race condition](https://github.com/teamcapybara/capybara/commit/4786328b84db0d7a43f0d85142d72fdf369639ee)
- (7c3d410) [Support held modifier keys in drag & drop](https://github.com/teamcapybara/capybara/commit/7c3d4107a5c276da8fdf49ce2d301d2e65b6fca9)
- (55f8304) [Minor fixups in drag modifiers and change option name](https://github.com/teamcapybara/capybara/commit/55f8304d1abcbb14b1a655db12dcdf4d71dfc675)


</details>